extensions/ql-vscode/src/distribution.ts

```diff
@@ -315,6 +315,15 @@ class ExtensionSpecificDistributionManager {
     const extensionSpecificRelease = this.getInstalledRelease();
     const latestRelease = await this.getLatestRelease();
 
+    // v2.12.3 was released with a bug that causes the extension to fail
+    // so we force the extension to ignore it.
+    if (
+      extensionSpecificRelease &&
+      extensionSpecificRelease.name === "v2.12.3"
```

Comment:
Minor:

```suggestion
      extensionSpecificRelease?.name === "v2.12.3"
```

Reply:
It _looks_ like at this point, we are guaranteed that 2.12.2 was not chosen since it is already being filtered out in `this.getLatestRelease()`. Is that true?

Reply:
`extensionSpecificRelease` is installed version though I think.

Reply:
Do you mean `2.12.2` or `2.12.3`?

I agree it should be the case that `latestRelease` cannot be `2.12.3` since it should be considered incompatible and get filtered out. Even if you're already on `2.12.3` it'll report `2.12.2` as the latest.

Reply:
> Do you mean 2.12.2 or 2.12.3?

Yes. :)